README.md

```diff
@@ -1,13 +1,104 @@
+# Klageskjema
+
+Frontend for klageskjema som legges ut på [klage.nav.no](https://klage.nav.no).
+
+## Hvordan kommer bruker seg til en digital klage?
+
+Det er to måter for en bruker å komme seg til en digital klage på.
+Enten via klageløsningens egen inngang eller sendt direkte til klageskjemaet fra en annen tjeneste.
+
+### Klageløsningen sin egen inngang
+
+Brukere kan navigere seg frem til riktig tema og tittel/ytelse via [klage.nav.no](https://klage.nav.no).
+Innlogging kreves ikke før bruker forsøker å gå til klageskjemaet.
+
+```
+https://klage.nav.no
+```
+
+### Direkte lenke
+
+Brukere kan være logget inn i en tjeneste som har nødvendig informasjon for å sende brukeren direkte til klageskjemaet, f.eks. en vedtaksløsning (selvbetjeningsløsning).
+
+1. Bruker ser vedtak i selvbetjeningsløsning.
+2. Bruker trykker på knappen `Klag på vedtak`.
+3. Bruker sendes til klageskjema.
+
+For å lenke direkte til klageskjemaet må `tema` og `tittel` være satt i URLen som query parametere.
+
+`tittel` må være en nøkkel som kan oversettes til tekst av klageløsningen.
+
+I tillegg til `tema` og `tittel` kan `saksnummer` settes på samme måte.
+`saksnummer` settes i klagen, men bruker kan **ikke** endre dette.
+
+Eksempel på en fullstendig URL til klageskjema:
+
+```
+https://klage.nav.no/ny?tema=FOR&tittel=NAV-LOVEN-14A&saksnummer=12345
```

Comment:
Skal vel også være underscore i stedet for dash, hilsen Flisespikker Spikkesen.

Reply:
Oh, ja. Litt vel kjapp på fiksingen der. Nå skal det være riktig.